beacon_node/execution_layer/src/engine_api/http.rs

```diff
@@ -784,6 +801,27 @@ impl HttpJsonRpc {
         Ok(response.into())
     }
 
+    pub async fn new_payload_v3<T: EthSpec>(
+        &self,
+        new_payload_request_deneb: NewPayloadRequestDeneb<T>,
```

Comment:
It might be ok to pass in the enum `NewPayloadRequest`, so we don't have to update this in the next fork?

Reply:
I missed this one, but we will resolve it next fork!